ethereum_history_api/circuits/lib/src/account_with_storage_int_test.nr

```diff
@@ -50,3 +52,13 @@ fn test_get_account_with_storage_invalid_state_root() {
     let storage_key = proofs[0].key;
     let _ = get_account_with_storage(number, address, storage_key);
 }
+
+#[test(should_fail)]
```

Comment:
Let's not make it normal to add a test without assertion and a comment. There should be a TODO comment here that this is like that only before your PR to noir-trie-proofs is fixed